src/App.js

```diff
@@ -45,11 +47,50 @@ const BuzzwordBingo = styled.div`
   background-color: #282c34;
 `;
 
+const Actions = styled.div`
+  display: flex;
+  justify-content: right;
+  width: 80vmin;
+  @media (max-width: 463px) {
+    width: 95vmin;
+  }
+`;
+
+const ActionButton = styled.button`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 4em;
+  width: 4em;
+  border: thin solid black;
+  margin: 0.5em;
+  border-radius: 5px;
+  &:hover {
+    scale: 1.05;
+  }
+  cursor: pointer;
+`;
+
+const RefreshButton = styled(ActionButton)`
+  background-color: lightblue;
+`;
+
 const App = () => {
+  const [refreshEvent, setRefreshEvent] = useState();
+
   return (
     <BuzzwordBingo>
       <h1>Buzzword Bingo</h1>
-      <Board buzzwords={buzzwords}></Board>
+      <Actions>
+        <RefreshButton type="button">
+          <BiRefresh
+            size={'5em'}
+            color="black"
+            onClick={(e) => setRefreshEvent(e.timeStamp)}
+          />
+        </RefreshButton>
+      </Actions>
+      <Board key={refreshEvent} buzzwords={buzzwords}></Board>
```

Comment:
How does that work?

Reply:
> Keys help React identify which items have changed, are added, or are removed.

If the `key` changes, React sees that item as removed + added, so it forces a re-render. [source](https://reactjs.org/docs/lists-and-keys.html#keys).

I might be abusing it though. It seems very much like a thing needed for lists specifically
> Keys should be stable, predictable, and unique. Unstable keys (like those produced by Math.random()) will cause many component instances and DOM nodes to be unnecessarily recreated, which can cause performance degradation and lost state in child components.

[source](https://reactjs.org/docs/reconciliation.html#tradeoffs)

Reply:
Another option would be to pull the `shuffledBuzzwords` logic up a level into the `App` component. Then the `App` passes a shuffled buzzwords list to `Board`

Reply:
> Another option would be to pull the `shuffledBuzzwords` logic up a level into the `App` component. Then the `App` passes a shuffled buzzwords list to `Board`

That would work. I think it would be cool to read the buzzwords from a static file or cookie at some point. That would help us implement #13.

Reply:
Cool I'll work on moving the `shuffledBuzzwords`